tests/integration/test_machine.py

```diff
@@ -36,3 +37,39 @@ async def test_status():
                          machine.status_message.lower() == 'running' and
                          machine.agent_status == 'started')),
             timeout=480)
+
+
+@base.bootstrapped
+async def test_scp(event_loop):
```

Comment:
This test seems redundant to me, as we already have it in the [intergration/test_unit.py](https://github.com/juju/python-libjuju/blob/16149dbef02160a7f4170e1630bfa069ba275479/tests/integration/test_unit.py#L160), and `unit.scp_to` is actually a wrapper around `machine.scp_to`, so it's not doing anything new or different (aside from the `add_machine` call I suppose).

Reply:
Removed the test - it is redundant indeed :) Thank you for the review!